README.md

```diff
@@ -242,3 +306,29 @@ class UserResource extends at.Resource {
 
 ***
 
+### Decorator 
+
+Angular provide a decorator feature to extends a service/factory. Use @decorator to use them and adding some others methods :
+
+```typescript
+@decorator('ngModuleName', 'SomeService')
+class SomeServiceDecorator {
+
+    constructor(
+        @at.inject('SomeService') private parent:SomeService;
+    ) {
```

Comment:
Please, drop constructor from this example as current implementation of `@decorator` will never inject anything into it.